emulator_test.go

```diff
@@ -341,22 +327,46 @@ func TestErrorTaskExecution(t *testing.T) {
 			},
 		},
 	}
+
+	start := time.Now()
+
 	createdTask, err := client.CreateTask(context.Background(), &createTaskRequest)
 	require.NoError(t, err)
 
+	// With the default retry backoff, we expect 4 calls within the first second:
+	// at t=0, 0.1, 0.3 (+0.2), 0.7 (+0.4) seconds (plus some buffer) ==> 4 calls
+	_, err = awaitHttpRequest(receivedRequests)
+	require.NoError(t, err)
+	log.Println("Got 1 request")
+
+	_, err = awaitHttpRequest(receivedRequests)
+	require.NoError(t, err)
+	log.Println("Got 2 requests")
+
+	_, err = awaitHttpRequest(receivedRequests)
+	require.NoError(t, err)
+
+	log.Println("Got 3 requests")
```

Comment:
did you intend to leave these printlns?

Reply:
Only sort-of, as I'd not-quite-registered that you could still pass an assertion message to `require.XXX` functions - have moved them there, they're only useful if it fails, to see what request we did/didn't get.